### DISCUSSION TITLE
Got stuck at the very beginning

### DISCUSSION THREAD

**Discussion by c0debreaker** [gruntwork-io/knowledge-base#730] [Help]
I'm on this page https://terratest.gruntwork.io/docs/getting-started/quick-start/
I'm so confused in the "MODULE NAME" from the command `go mod init "<MODULE_NAME>"`

It says
```
Where <MODULE_NAME> is the name of your module, typically in the format github.com/<YOUR_USERNAME>/<YOUR_REPO_NAME>.
```
However, what would be the value for "YOUR_REPO_NAME"? It's confusing since I just wanted to test their example.

Hmm, maybe the url link I found is not for learning how to use terratest.

**Comment by gitsstewart** [comment]
@c0debreaker the guidance provided is intended to help you download and start using terratest for yourself in your own projects. Therefore, you would in theory have your own repository that contains the modules that you have built. We don't provide a "demo" of terratest that you can run from our repository - you have to follow the instructions to first make a local copy for yourself, that local copy would be in your repository and therefore the repo where you have that copy corresponds to "YOUR_REPO_NAME"

**Reply by c0debreaker** [reply]
Ah, I see. 

You wrote this
```
you would in theory have your own repository that contains the modules that you have built. 
```
When you say modules, are you referring to the Terraform .tf files I have created?

**Reply by tracphil** [reply]
@c0debreaker @gitsstewart is referring to [Terraform Modules](https://developer.hashicorp.com/terraform/language/modules).

**Reply by c0debreaker** [reply]
> @c0debreaker @gitsstewart is referring to [Terraform Modules](https://developer.hashicorp.com/terraform/language/modules).

Got it. Thanks a lot!
